context/selection-of-signal-for-I.md

```diff
@@ -0,0 +1,165 @@
+        * With market based reporting organisations now have two ways to report a lower scope 2 emissions figure to the public.(1) Consume less energy by making your processes more efficient. (2) Consume the same energy and purchase certificates. (2) is the easier solution for an organisation to prioritise than (1) which takes significant investment and effort. 
+        * Renewable energy purchasing is often managed by a different team than the teams that develop software, which means organisations can reduce their emissions through market based measures without engaging in any efficiency improvement activity in the software products.
+        * In addition if an organisation is already reporting 0 KT in terms of Scope 2 because of their purchase of energy certificates, and did more work to make their processes more energy efficient, they would still report 0 KT, a metric that stays at 0 regardless of what you do, does not encourage action..
+        * Market based reporting doesn’t incentivize making applications more energy efficient.
+
+
+## References
+
+_List here any references to external sources, plus any specific insight from the reference which would aid in this discussion topic._
+
+* SRMER are very good at assessing the short-term impact of an intervention
+    * Source: [How Well Do Emission Factors Approximate Emission Changes from Electricity System Models?](https://pubs.acs.org/doi/abs/10.1021/acs.est.2c02344)
+    * Twitter Discussion about it: [Twitter Thread from Eric Hittinger](https://twitter.com/ElephantEating/status/1590741977880211458)
+    * Twitter Discussion providing a contrary view: [Twitter Thread from AukeHoekstra responding to the Twitter Thread by Eric Hittinger](https://twitter.com/AukeHoekstra/status/1620525739803291648) With clarification for load shifting: [https://twitter.com/Helmut_Lehmeyer/status/1620766545793404928](https://twitter.com/Helmut_Lehmeyer/status/1620766545793404928)
+* LRMER are very strong predictors of long-term impacts of interventions. AER are a strong proxy for LRMER, while SRMER does not capture long-term impacts of interventions
+    * [Planning for the evolution of the electric grid with a long-run marginal emission rate](https://www.cell.com/iscience/fulltext/S2589-0042(22)00185-7?_returnURL=https%3A%2F%2Flinkinghub.elsevier.com%2Fretrieve%2Fpii%2FS2589004222001857%3Fshowall%3Dtrue)
+    * [Short-run marginal emission rates omit important impacts of electric-sector interventions](https://www.pnas.org/doi/10.1073/pnas.2211624119)
+* Marginal emission factors are based on estimation models and their value cannot be verified. This is because there is no “ground truth” on the true value as the specific marginal generator cannot be verified. But there is good academic research showing that marginal emissions can be calculated and are a meaningful signal.

```

Comment:
I think this considerably overstates the uncertainly around marginal emissions and how to calculate them. There's further consensus in the academic literature that marginal emissions are the correct metric for measuring the impact of interventions.

Reply:
The above statements are based on the insights provided by the cited reports and discussions with US and EU TSOs. Moreover, the fact that SRMER have not been verified has also been raised in today's session by Pieter Gagnon. 

Inside the original context document, right after this line there are many sources. For convenience, I have provided them here again. As a reminder, these are **cited statements**, i.e. these are exact comments from these reports: 

-  [The US RTO PJM states that “marginal units – and the marginal emissions rates based on them –, cannot provide any prediction of the results of an action. "](https://pjm.com/-/media/etools/data-miner-2/marginal-emissions-primer.ashx)
- [“All marginal emissions rate estimation methodologies are imperfect for one reason or another”](https://media.rff.org/documents/Report_22-08.pdf)
- [“There is tremendous variation in the way that MEFs can be estimated. Depending on the specific methodology, data sources, and assumptions used, no one estimate is likely to match another. Even two approaches using the same methodology could use different input data or assumptions to estimate marginal emissions impact, leading to two different and sometimes inconsistent estimates. This means that there is no single source of “truth” for marginal emissions impact.”](https://cebi.org/wp-content/uploads/2022/11/Guide-to-Sourcing-Marginal-Emissions-Factor-Data.pdf)

Reply:
Here are a few published peer reviewed academic papers that discuss marginal emissions as being strongly supported science: 

“We compare marginal and average emissions factors, finding that AEFs may grossly misestimate the avoided emissions resulting from an intervention… Both supply- and demand-side interventions will displace energy and emissions from conventional generators. MEFs give a consistent metric for assessing the avoided emissions resulting from such interventions.” — [Stanford’s Azevedo et al. in Environmental Science & Technology (2012)](https://pubmed.ncbi.nlm.nih.gov/22486733/)

“These [marginal emissions] estimates have important implications for understanding the environmental consequences of many electricity-shifting policies… [they] are also relevant for understanding the impacts of activities and policies that increase electricity demand.” — [Yale’s Kotchen et al. in the Journal of Economic Behavior & Organization (2014)](https://faculty.tuck.dartmouth.edu/images/uploads/faculty/erin-mansur/mansur_marginal_emissions.pdf)

“If the load under analysis represents an incremental increase or decrease, marginal ERs can provide an emissions measurement specific to the change. Thus, for consequential LCAs of new loads, marginal factors are recommended.” — [University of Michigan’s Ryan et al. in Environmental Science & Technology (2016)](https://pubs.acs.org/doi/10.1021/acs.est.5b05216)

“...the use of AEFs, which reflect grid-average situations, to estimate the effect of an intervention may be problematic, because not all generating technologies would respond to changes in demand proportionally… AEFs could significantly misestimate the amount of emissions avoided by an intervention. By contrast, MEFs estimate the emission intensity of marginal power generation that responds to a change in demand, and are a more appropriate metric to assess emission implications of policy and technology interventions.” — [University of Minnesota’s Smith et al. in Environmental Science & Technology (2017)](https://pubs.acs.org/doi/10.1021/acs.est.7b00034)

“Increasing (or decreasing) demand at a given time affects the operation of only the marginal units in the dispatch order, and not the infra-marginal units. Therefore, emission implications of energy storage, or any other incremental demand- or supply-side resource that leads to load-shifting, depend on marginal operating emission rates, based on the emission intensity of the marginal units, and not the average operating emission rates, based on the average emission intensity of the entire grid.” — [Columbia’s Shrader et al. in SSRN: Social Science Research Network (2020)](https://papers.ssrn.com/sol3/papers.cfm?abstract_id=3502271)

“Marginal Emission Rate provides a mathematically sound and transparent way to quantify the carbon footprint of electricity consumption and production. Mathematically, the MER calculation is similar to the calculation of Locational Marginal Prices (LMP) and system lambda, which are used for economic dispatch of power systems across North America.” — [Tabors Caramanis Rudkevich’s Hua He et al. in The Electricity Journal (2021)](https://docs.google.com/document/d/1yIHjDCoQQtrBUx570aQhXnaVCKGz8-ZyOxGJb_kup78/edit#)